### DISCUSSION TITLE
soft-wrap to ignore newline (line-ending)

### DISCUSSION THREAD

**Discussion by cobratbq** [helix-editor/helix#9791] [Ideas] [answered]
Hi, I tried searching but did not found a similar configuration option/issue/discussion on the topic.

Has it been considered to ignore the newline (line-ending) char when (soft-)wrapping? I find that, all the times when the (soft-)wrap initiates when only the newline is left, it leaves me surprised. In a way, if the newline is the only char left, then the start of the next line is exactly the intended rendering. I come across this case quite regularly and it seems off.

Note that I am aware of the position to render the newline as any other character, and that's fine, so my question is specifically for (soft-)wrapping. Also, `g`,`l` to go to line-end also ignores newline character.

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/issues/6765

**Reply by cobratbq** [reply]
well, I searched for the wrong keywords then :-/
Thanks for the quick follow-up.
